### ISSUE TITLE
monero-gui-osx-10.11 build crash

### ISSUE THREAD

**Issue by Afron-Lysias** [monero-project/monero-gui#2139]
For the last few weeks I have been testing monero-gui-osx-10.11 builds on a mac without brew (on purpose). 
Any build newer than #1682 crashes for the following reason:

Dyld Error Message:   Library not loaded: /usr/local/Cellar/openssl/1.0.2q/lib/libcrypto.1.0.0.dylib

Since this was a problem that created a lot of confusion during the previous version update, I open this issue as an alert to make shure that this time we can avoid similar mistakes in the final version.

**Comment by selsta** [comment]
Could you check https://build.getmonero.org/downloads/monero-wallet-gui-ff6ce62-osx-10.12.zip ?

**Comment by Afron-Lysias** [comment]
This is an osx-10.12 build. I'm testing exclusively osx-10.11 builds. 
I have tested ALL new osx-10.11 builds since #1682.

Test Results for monero-wallet-gui-ff6ce62-osx-10.12:
Application: Runs
Wallet: Opens
Daemon: Failed to start

`[RPC0]	ERROR	net.ssl	contrib/epee/src/net_ssl.cpp:388	SSL handshake failed, connection dropped: no shared cipher
 `

**Comment by xiphon** [comment]
@Afron-Lysias 
 
> Daemon: Failed to start

Did you try to start `monerod` with `monero-wallet-gui-ff6ce62-osx-10.12.zip` and it crashed right after the start?

**Comment by Afron-Lysias** [comment]
Yes, I started monerod within monero-wallet-gui-ff6ce62-osx-10.12. 
GUI itself didn't crash. The daemon just failed to start.

**Comment by dEBRUYNE-1** [comment]
Could you open a separate issue for that daemon failed to start issue on the main repository (i.e. https://github.com/monero-project/monero/issues)?

**Comment by Afron-Lysias** [comment]
Regarding "monero-wallet-gui-ff6ce62-osx-10.12" problem, aren't issues expected, since I'm running an older macOS (10.11) than required, according to the file name?

**Comment by xiphon** [comment]
> Regarding "monero-wallet-gui-ff6ce62-osx-10.12" problem, aren't issues expected, since I'm running an older macOS (10.11) than required, according to the file name?

Nope. is static build, there was a good chance it might get working on 10.11. Usually OS dumps some error message and details in the console output like seg fault or library loading error, for example.

Anyway working on static 10.11 build now. Will let you know once it is ready.

**Comment by Afron-Lysias** [comment]
Here it is:
https://github.com/monero-project/monero-gui/issues/2159

**Comment by xiphon** [comment]
@Afron-Lysias 
Please check this one https://build.getmonero.org/downloads/monero-wallet-gui-3c1fe1da-osx-10.11.zip

**Comment by Afron-Lysias** [comment]
@xiphon 
This version is OK. Everything is back to normal, plus the white light theme.

**Comment by Afron-Lysias** [comment]
This version has also fixed this problem:
https://github.com/monero-project/monero-gui/issues/1459

**Comment by selsta** [comment]
+resolved

**Comment by selsta** [comment]
+resolved
